package.json

```diff
@@ -363,7 +363,7 @@
     "tslint": "^5.8.0",
     "tslint-microsoft-contrib": "^5.0.1",
     "typescript": "^2.9.2",
-    "vscode": "^1.1.21"
+    "vscode": "^1.1.22"
```

Comment:
why need to update vscode?

Reply:
the dependency is introduced by vscode.
```bash
$> npm ls event-stream
vscode-maven@0.11.2 /Users/sechs/Documents/vscode-maven
└─┬ vscode@1.1.21
  ├─┬ gulp-remote-src-vscode@0.5.0
  │ └── event-stream@3.3.5
  ├─┬ gulp-symdest@1.1.0
  │ └── event-stream@3.3.5  deduped
  ├─┬ gulp-untar@0.0.7
  │ └── event-stream@3.3.5  deduped
  └─┬ gulp-vinyl-zip@2.1.0
    └── event-stream@3.3.5  deduped

```

Reply:
Is event-stream an implicit dependency of vscode package?